doc/intl/de_GER/README.md

```diff
@@ -80,25 +70,13 @@ Falls Sie einen Fehler oder Probleme beim Verwenden dieser Software finden, meld
 Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind, sehen Sie sich bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
 Oft gibt es Themen, die [Hilfe](https://github.com/dogecoin/dogecoin/labels/help%20wanted) benötigen. Ihr Beitrag könnte einen hohen Stellenwert haben und wird daher auch sehr geschätzt.
  
-## Communities 🚀🍾
- 
-Sie können den Communities auf verschiedenen sozialen Medien beitreten.
-Herzlich sind Sie eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
-Vielleicht findet sich ja auch jemand, der Ihnen bei Ihren Projekten behilflich sein kann.
- 
-Hier sind einige informative Links, die Sie interessieren könnten:
- 
-* [Dogecoin subreddit](https://www.reddit.com/r/dogecoin/)
-* [Dogeducation subreddit](https://www.reddit.com/r/dogeducation/)
-* [Discord](https://discord.gg/dogecoin)
-* [Dogecoin Twitter](https://twitter.com/dogecoin) 
- 
 ## Oftmals gestellte Fragen: ❓
  
-Haben Sie eine Frage zu Dogecoin? Eine Antwort könnte vielleicht bereits in der [FAQ](doc/FAQ.md) oder im
+Haben Sie eine Frage zu Dogecoin? Eine Antwort könnte vielleicht bereits in der
+[FAQ](doc/FAQ.md) oder im
 [Q&A](https://github.com/dogecoin/dogecoin/discussions/categories/q-a) hier auf Github vorhanden sein!
  
 ## Lizenz ⚖️
  
-Dogecoin Core wird unter den Bedingungen der MIT-Lizenz veröffentlicht.
-Siehe: [COPYING](COPYING) für mehr Informationen oder informiere Dich bei [opensource.org](https://opensource.org/licenses/MIT).
+Dogecoin Core wird unter den Bedingungen der MIT-Lizenz veröffentlicht. Siehe
+[COPYING](COPYING) für mehr Informationen.
```

Comment:
81 - 82 :
In general fine but " Siehe" -imho- deffo belongs into the next line here, like in old line 103.
short expl. as to why: as it is the start of a new Sentence, which should start in a new line..
but still: great work on that!

Reply:
Yes, I tried to standardize it, but I should have asked a German to confirm. Good point, I'll return the sentence to its original state.